src/v2/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -1,14 +1,190 @@
+        } else if (!!editAlertEntity && !isCurrentEdgeSelected) {
+          variant = "inactive"
+        }
+
+        return (
+          <Fragment key={edge.internalID}>
+            <SavedSearchAlertListItemFragmentContainer
+              item={edge}
+              variant={variant}
+              onEditAlertClick={setEditAlertEntity}
+            />
+            {!isLastEdge && <Separator color="black15" />}
+          </Fragment>
+        )
+      })}
+    </>
+  )
 
-export const SavedSearchAlertsApp: React.FC = ({ children }) => {
   return (
     <Box>
-      <MetaTags title="Your Alerts | Artsy" pathname="/alerts" />
-      <Text variant="xl" my={4}>
-        Your Alerts
-      </Text>
-      {children}
+      <MetaTags title="Your Alerts | Artsy" pathname="/user/alerts" />
+
+      <Media greaterThanOrEqual="md">
+        <SavedSearchAlertHeader />
+      </Media>
+      <Media lessThan="md">{!isEditMode && <SavedSearchAlertHeader />}</Media>
+
+      <FullBleed>
+        <Media greaterThanOrEqual="md">
+          <Separator color="black15" />
+        </Media>
+
+        <Media lessThan="md">
+          {!isEditMode && <Separator color="black15" />}
+        </Media>
+
+        {alerts.length === 0 ? (
+          <SavedSearchAlertsEmptyResults />
+        ) : (
+          <>
+            <Media greaterThanOrEqual="md">
+              <GridColumns gridColumnGap={0}>
+                <Column span={isEditMode ? 6 : 12}>{list}</Column>
+                {isEditMode && editAlertEntity && (
+                  <Column span={6}>
+                    <SavedSearchAlertEditFormContainer
+                      editAlertEntity={editAlertEntity}
+                      onCloseClick={handleCloseClick}
+                      onCompleted={handleCompleted}
+                      onDeleteClick={handleDeleteClick}
+                    />
+                  </Column>
+                )}
+              </GridColumns>
+            </Media>
+
+            <Media lessThan="md">
+              {isEditMode && editAlertEntity ? (
+                <SavedSearchAlertEditFormContainer
+                  editAlertEntity={editAlertEntity}
+                  onCloseClick={handleCloseClick}
+                  onCompleted={handleCompleted}
+                  onDeleteClick={handleDeleteClick}
+                />
+              ) : (
+                list
+              )}
+            </Media>
+
+            {showDeleteModal && editAlertEntity && (
+              <SavedSearchAlertDeleteModal
+                id={editAlertEntity!.id}
+                onCloseClick={handleCloseDeleteModal}
+                onDeleted={handleDeleted}
+              />
+            )}
+          </>
+        )}
+      </FullBleed>
     </Box>
   )
 }
+
+export const SavedSearchAlertsAppPaginationContainer = createPaginationContainer(
+  SavedSearchAlertsApp,
+  {
+    me: graphql`
+      fragment SavedSearchAlertsApp_me on Me {
+        savedSearchesConnection(first: 50)
```

Comment:
**question**: I see that this is PaginationContainer but I don't see any Pagination controls yet and this `first` number seems high. Are we following up with those pagination controls in a separate ticket/PR?

Reply:
> Are we following up with those pagination controls in a separate ticket/PR?

Yes, I think it would be better to add these changes in a separate PR